docs/guides/es2k/deploying-p4-programs.md

```diff
@@ -42,7 +42,7 @@ root@mev-imc:/etc/dpcp/package# cp simple_l3_l4_pna.pkg default_pkg.pkg
 ```
 
 If COMMS channel support is required, follow
```

Comment:
```suggestion
If Communication Channel support is required, follow
```

Reply:
I'm inclined to take the opposite tack and change "communication channel" to "COMMS" channel.

Reply:
I changed it to Communication Channel, since that's what IMC team uses in their release notes and other documentation and would like to keep it consistent all across

Reply:
I've update the document to use Communication Channel instead of COMMS.